CONTRIBUTING.rst

```diff
@@ -36,22 +36,27 @@ can be contributed to scikit-image.
      <https://github.com/scikit-image/scikit-image>`_ and click the
      "fork" button to create your own copy of the project.
 
+   * `Set up GitHub SSH authentication <https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh>`__.
+
    * Clone (download) the repository with the project source on your local computer::
 
-      git clone https://github.com/your-username/scikit-image.git
+      git clone git@github.com:scikit-image/scikit-image
```

Comment:
Following my other [comment](https://github.com/scikit-image/scikit-image/pull/7953/files#r2550393431)
 
```suggestion
      git clone --origin upstream git@github.com:scikit-image/scikit-image
```
and along what @matthew-brett suggests.